core/templates/dev/head/services/CsrfTokenServiceSpec.ts

```diff
@@ -50,6 +47,8 @@ describe('Csrf Token Service', function() {
   });
 
   it('should error if getTokenAsync is called before initialize', () => {
+    csrfTokenService.getTokenAsync();
```

Comment:
Why are you adding this? This will throw `'Token needs to be initialized'`.

Reply:
was trying all the possible permutations! Since i can't get it to pass.